Source/WebKit/UIProcess/RemoteLayerTree/RemoteLayerTreeInteractionRegionLayers.mm

```diff
@@ -114,7 +114,7 @@ void updateLayersForInteractionRegions(CALayer *layer, const RemoteLayerTreeTran
                     [interactionRegionLayer setBackgroundColor:cachedCGColor({ WebCore::SRGBA<float>(0, 1, 0, .3) }).get()];
 
                 setInteractionRegion(interactionRegionLayer.get(), region);
-                configureLayerForInteractionRegion(interactionRegionLayer.get(), @"WKInteractionRegion");
+                configureLayerForInteractionRegion(interactionRegionLayer.get(), String::number(region.elementIdentifier.toUInt64()));
```

Comment:
Hm...it seems like it would still be useful to prefix this with "WKInteractionRegion-<n>" (assuming that we'll use this string to set the layer name?)

Would something like `makeString("WKInteractionRegion-"_s, String::number(region.elementIdentifier.toUInt64()))` sound reasonable?

Reply:
Not actually the layer name (though... we should totally name these too, that would be handy -- maybe even with element/renderer info in debug), but I think your suggestion still makes total sense.